### PR TITLE
Update a product from the Android mobile app

### PR DESCRIPTION
**Pull request by teolemon**
Update a product from the Android mobile app

https://world.openfoodfacts.net/update-a-product-from-the-mobile-app

### CODE REVIEW

File: lang/en/texts/update-a-product-from-the-mobile-app.html [hunk 1]
```diff
@@ -0,0 +1,666 @@
+<h1>Update a product from the mobile app</h1>
+
+<div class="row">
+	<div class="small-12 medium-4 columns">
+        <img src="https://upload.wikimedia.org/wikipedia/commons/1/17/FAQ_icon_%28Noun_like%29.svg">
```
Comment (hangy) [inline lang/en/texts/update-a-product-from-the-mobile-app.html:5]: Can we use this without attribution?

File: lang/en/texts/update-a-product-from-the-mobile-app.html [hunk 2]
```diff
@@ -0,0 +1,666 @@
...
+        <div class="card-section">
+          <h4><a href="/faq">Frequently Asked Questions</a></h4>
+          <p>It has an easy to override visual style, and is appropriately subdued.</p>
```
Comment (hangy) [inline lang/en/texts/update-a-product-from-the-mobile-app.html:8]: What does the text mean? It's repeated several times.

File: lang/en/texts/update-a-product-from-the-mobile-app.html [hunk 3]
```diff
@@ -0,0 +1,666 @@
...
+	<figure>
+	    <img src="/images/misc/mobile-app-help/android-fr-robotoff-question-result.png"
+	         alt="">
+	    <figcaption>Thanks to the category you have just validated by answering the question, Open Food Facts was able to compute the Nutri-Score of the product.</figcaption>
+	</figure>
+</div>
+</div>
+
+<!-- Ingredients -->
+<h3 style="background-color:#09f;padding:1%;color:#fff;font-weight:bold;">1. Recropping a previously uploaded photo to add a missing ingredient picture</h3>
```
Comment (hangy) [inline lang/en/texts/update-a-product-from-the-mobile-app.html:90]: Why is every heading "1."?

File: lang/en/texts/update-a-product-from-the-mobile-app.html [hunk 4]
```diff
@@ -0,0 +1,666 @@
...
+          <h4><a href="/update-a-product-from-the-mobile-app">How to complete products on Android</a></h4>
+          <p>It has an easy to override visual style, and is appropriately subdued.</p>
+        </div>
+	</div>
+    
+</div>
+
+
+<!-- Robotoff -->
+<h3 style="background-color:#09f;padding:1%;color:#fff;font-weight:bold;">1. Answer questions to help complete products</h3>
```
Comment (hangy) [inline lang/en/texts/update-a-product-from-the-mobile-app.html:62]: `h1` should be followed by `h2`

Comment (hangy) [reply lang/en/texts/update-a-product-from-the-mobile-app.html:62]: Style is repeated several times. Move to scss class?

File: lang/en/texts/update-a-product-from-the-mobile-app.html [hunk 5]
```diff
@@ -0,0 +1,666 @@
...
+<div class="small-12 medium-6 columns">
+	<figure>
+	    <img src="/images/misc/mobile-app-help/android-fr-take-a-photo.png"
+	         alt="">
+	    <figcaption>Due to the collaborative nature of open food facts,  some photos may be missing.  just click on the take photo icon to take part in food transparency.</figcaption>
+	</figure>
+</div>
+</div>
+
+<p class="callout panel">You need the Android Open Food Facts app to update a product that way. You can get it <a href="https://android.openfoodfacts.org/">there</a>.</figcaption>
```
Comment (hangy) [inline lang/en/texts/update-a-product-from-the-mobile-app.html:219]: What about the website or iOS app? You don't *need* the Android app …

File: lang/en/texts/update-a-product-from-the-mobile-app.html [hunk 6]
```diff
@@ -0,0 +1,666 @@
...
+
+<style type="text/css">
```
Comment (hangy) [inline lang/en/texts/update-a-product-from-the-mobile-app.html:221]: Move to minified scss?

File: lang/en/texts/update-a-product-from-the-mobile-app.html (hunk 1 above)
Comment (teolemon) [reply lang/en/texts/update-a-product-from-the-mobile-app.html:5]: no

File: lang/en/texts/update-a-product-from-the-mobile-app.html (hunk 2 above)
Comment (teolemon) [reply lang/en/texts/update-a-product-from-the-mobile-app.html:8]: it's placeholder text :-)

File: lang/en/texts/update-a-product-from-the-mobile-app.html (hunk 3 above)
Comment (teolemon) [reply lang/en/texts/update-a-product-from-the-mobile-app.html:90]: the structure is not final

File: lang/en/texts/update-a-product-from-the-mobile-app.html (hunk 5 above)
Comment (teolemon) [reply lang/en/texts/update-a-product-from-the-mobile-app.html:219]: this tutorial is Android only at this point. We don't have such advanced features in the iOS app

**Comment by teolemon** [comment]
https://github.com/openfoodfacts/openfoodfacts-web/pull/1

**Pull request closed without merging**